modules/recommended-configurable-storage-technology.adoc

```diff
@@ -51,7 +51,7 @@ The following table summarizes the recommended and configurable storage technolo
 
 ^5^ For metrics, using file storage with the `ReadWriteMany` (RWX) access mode is unreliable. If you use file storage, do not configure the RWX access mode on any persistent volume claims (PVCs) that are configured for use with metrics.
 
-^6^ For logging, using any shared storage would be an anti-pattern. One volume per elasticsearch is required.
+^6^ For logging, review the recommended storage solution in configuring persistent storage for the log store section. Using NFS storage as a persistent volume or through NAS such as Gluster can corrupt the data. Hence, NFS is not supported for Elasticsearch storage and LokiStack log store in {product-title} Logging. You must use one persistent volume type per log store.
```

Comment:
Just w
```suggestion
^6^ For logging, review the recommended storage solution in configuring persistent storage for the log store section. Using NFS storage as a persistent volume or through NAS, such as Gluster, can corrupt the data. Hence, NFS is not supported for Elasticsearch storage and LokiStack log store in {product-title} Logging. You must use one persistent volume type per log store.
```

Reply:
I'm wondering if the C in Configuring persistent storage needs to be capitalized, since you're referring to the title of a piece of docs.

Reply:
That makes sense- Addressed